app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -72,43 +73,58 @@ keywords:
 ---
 $applications-open$
 
-If you are a teacher considering moving to England to teach languages or physics, you could be eligible for financial help from the UK government worth £10,000.
+## What the IRP is for
+
+If you are a teacher considering moving to England to teach languages or physics, you could be eligible for an international relocation payment (IRP) from the UK government worth up to £10,000.
 
-The international relocation payment (IRP) is designed to cover some of the costs of moving to England, including:
+The IRP is designed to cover some of the costs of moving to England, including:
 
 * the costs of visas
 
 * the immigration health surcharge
 
 * other relocation expenses
 
-The IRP is a grant – you will not have to pay this money back. If you are eligible, you can only receive the IRP once.
+You can apply for the IRP between September 2024 and March 2025. If you are eligible, you will receive a maximum IRP of £10,000, in 2 instalments of £5,000 spread over 2 years.
+
+The IRP is available whether you teach full-time or part-time.
```

Comment:
The IRP is a 2 year trial to support non-UK citizen who want to teach in England. 2024 to 2025 is the second year of the trial.